restless/dj.py

```diff
@@ -1,7 +1,15 @@
 import six
 
+from django import get_version as get_django_version
 from django.conf import settings
-from django.conf.urls import patterns, url
+
+if get_django_version() >= '1.10.0':
```

Comment:
This comparison might fail in some circumstances. Please see http://stackoverflow.com/questions/11887762/compare-version-strings